dora/core/client/fs/src/main/java/alluxio/client/file/DoraCacheFileSystem.java

```diff
@@ -379,6 +380,18 @@ public void rename(AlluxioURI src, AlluxioURI dst, RenamePOptions options)
 
       mDoraClient.rename(srcUfsFullPath.toString(), dstUfsFullPath.toString(), mergedOptions);
     } catch (RuntimeException ex) {
+      if (ex instanceof StatusRuntimeException) {
+        Status.Code code = ((StatusRuntimeException) ex).getStatus().getCode();
+        if (code == Status.FAILED_PRECONDITION.getCode()) {
+          // throw new RuntimeException(ex.getMessage());
```

Comment:
I just wonder why comment this line.

Reply:
just not sure this is the appropriate exception to throw